### PR TITLE
🐛Only expand strings in arrays

### PR DESCRIPTION
**Pull request by micajuine-ho**
Follow up to #26157

### CODE REVIEW

File: extensions/amp-analytics/0.1/variables.js [hunk 1]
```diff
@@ -281,7 +281,10 @@ export class VariableService {
       } else if (isArray(value)) {
         // Treat each value as a template and expand
         for (let i = 0; i < value.length; i++) {
-          value[i] = this.expandValue_(value[i], options);
+          value[i] =
+            typeof value[i] == 'string'
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/variables.js]: would it ever be the case where `value[i]` is an object or an array? Or anything that's not a number or a boolean?

Comment (micajuine-ho) [reply extensions/amp-analytics/0.1/variables.js]: @zhouyx We discussed in #26157, how arrays inside arrays would not be supported (unless they it is requested). I think the same approach should apply for objects. 

Currently, our documentation does not say anything about objects, (e.g. they will just get url-encoded), and after testing, I can confirm that it does not break on objects.

Comment (zhouyx) [reply extensions/amp-analytics/0.1/variables.js]: Right we don't want to support expanding nested array. Just to confirm that a nested array won't break the usage.

Comment (micajuine-ho) [reply extensions/amp-analytics/0.1/variables.js]: Confirmed. It does not break the usage. Should I add a test example?

Comment (zhouyx) [reply extensions/amp-analytics/0.1/variables.js]: nah, should be good : )

**Pull request merged**